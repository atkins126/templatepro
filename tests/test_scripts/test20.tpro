DataSets
{{loop(customers)}}
Code: {{:customers.Code}} - Name: {{:customers.Name}}
{{endloop}}

List of objects
{{loop(objects)}}
Prop1: {{:objects.Prop1}} {{:value2}}
---
{{endloop}}
