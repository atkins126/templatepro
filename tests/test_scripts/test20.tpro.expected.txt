DataSets
Code: 1 - Name: Ford
Code: 2 - Name: Ferrari
Code: 3 - Name: Lotus
Code: 4 - Name: FCA
Code: 5 - Name: Hyundai
Code: 6 - Name: De Tomaso
Code: 7 - Name: Dodge
Code: 8 - Name: Tesla
Code: 9 - Name: Kia
Code: 10 - Name: Tata
Code: 11 - Name: Volkswagen
Code: 12 - Name: Audi
Code: 13 - Name: Skoda

List of objects
Prop1: value1.1 DANIELE2
---
Prop1: value1.2 DANIELE2
---
Prop1: value1.3 DANIELE2
---
